scripts/tcs_startup.sh

```diff
@@ -19,8 +19,18 @@ listener="${TCF_HOME}/examples/common/python/connectors/direct/tcs_listener/tcs_
 # Read Listener port from config file
 listener_port=`grep listener_port ${TCF_HOME}/config/tcs_config.toml | awk {'print $3'}`
 
+lmdb_server="${TCF_HOME}/examples/common/python/shared_kv/remote_lmdb/lmdb_listener.py"
+# the following port must be synchronized to that of KvStorage.database_url 
+# in config/tcs_config.toml
+LMDB_SERVER_PORT=9090
```

Comment:
Can you read the LMDB_SERVER_PORT from config file similar to listener_port?

Reply:
done cd711736a8b2b953501e3fe18a4a2680bbb496a6